ansible/playbook/tasks/update.yml

```
@@ -182,7 +182,7 @@
         state: latest
         security: yes
         exclude:
-          - nginx
+          - 'nginx*'
```

```suggestion
          - "nginx*"
```
Seems like we use double quotes everywhere.

---

I see, but then it's not consistent with the rest of the code in the same file. Do you want to replace double quotes with single quotes everywhere?